packages/contracts/scripts/deploy-scripts/mainnet.sh

```diff
@@ -47,22 +19,21 @@ fi
 
 CONTRACTS_TARGET_NETWORK=mainnet \
 npx hardhat deploy \
- --ctc-force-inclusion-period-seconds 12592000 \
- --ctc-max-transaction-gas-limit 11000000 \
- --em-max-gas-per-queue-per-epoch 250000000 \
- --em-max-transaction-gas-limit 11000000 \
- --em-min-transaction-gas-limit 50000 \
- --em-ovm-chain-id 10 \
- --em-seconds-per-epoch 0 \
  --l1-block-time-seconds 15 \
- --ovm-address-manager-owner 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A \
- --ovm-proposer-address 0x473300df21D047806A082244b417f96b32f13A33 \
- --ovm-relayer-address 0x0000000000000000000000000000000000000000 \
- --ovm-sequencer-address 0x6887246668a3b87F54DeB3b94Ba47a6f63F32985 \
- --reset \
+ --ctc-max-transaction-gas-limit 15000000 \
+ --ctc-l2-gas-discount-divisor 32 \
+ --ctc-enqueue-gas-cost 60000 \
  --scc-fraud-proof-window 604800 \
  --scc-sequencer-publish-window 12592000 \
+ --ovm-sequencer-address 0x6887246668a3b87F54DeB3b94Ba47a6f63F32985 \
+ --ovm-proposer-address 0x473300df21D047806A082244b417f96b32f13A33 \
+ --ovm-address-manager-owner 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A \
+ --gasprice 150000000000 \
```

Comment:
This may not be the best value to use at the time, will need to determine what it should be at deploy time

Reply:
This could be determined at runtime using `seth` but unsure if we want to add another dependency to this script

Reply:
`seth basefee` or `seth gas-price`